src/Bridge/Symfony/DependencyInjection/NelmioAliceExtension.php

```diff
@@ -70,6 +70,8 @@ private function loadServices(ContainerBuilder $container)
             );
         }
 
-        $container->registerForAutoconfiguration(BaseFakerProvider::class)->addTag('nelmio_alice.faker.provider');
+        if ( 0 === count($container->findTaggedServiceIds('nelmio_alice.faker.provider')) ) {
```

Comment:
Could you add the following comment? just above it?

```
// Check if the auto-configuration of the tag is not already done. This is a temporary measure to avoid to break
// existing versions of HautelookAliceBundle
```